cookbook/routing/extra_information.rst

```diff
@@ -52,12 +52,16 @@ arguments to your controller:
 
         return $collection;
 
-Now, you can access this extra parameter in your controller::
+Now, you can access this extra parameter in your controller, either as an argument (if specified), or through the ``Request`` object::
 
-    public function indexAction($page, $title)
+    use Symfony\Component\HttpFoundation\Request;
+    
+    public function indexAction(Request $request, $page, $title)
     {
         // ...
+        $titleAttribute = $request->attributes->get('title'); // same as $title
```

Comment:
`$titleAttribute` is confusing imo, to show this, its enough to write `$request->attributes->get('title'); // same as $title`

what do you thing @javiereguiluz